perl/meson_options.txt

```diff
@@ -0,0 +1,50 @@
+# Nix-Perl build options
+#============================================================================
+
+
+# compiler args
+#============================================================================
+
+option(
+    'version',
+    type : 'string',
+    description : 'nix-perl version')
+
+option(
+    'tests',
+    type : 'feature',
+    value : 'disabled',
+    description : 'run nix-perl tests')
```

Comment:
Meson doesn't have an automatic version of this? (I forget)

Reply:
No, I’ve argued for one but other people don’t like the idea. Maybe I should just write some code